assets/app/styles/_components.less

```diff
@@ -302,13 +302,8 @@
         }
 
         .pod-status-label {
-          &:after {
-            content: "\e608"; // pficon-error-exclamation
-            left: -16px;
-            top: 0;
-          }
           &:before {
-            content: "\e609"; // pficon-error-octagon
+            content: "\e61c"; // pf-warning-triangle-o
```

Comment:
The error icon, octagon & !, was 2 icons layered together (e608 and e609). Those were removed and replace with a single icon triangle & ! (e61c)